azure-pipelines-install-node.yml

```diff
@@ -0,0 +1,92 @@
+# Template file to install node using NVM on linux and manual install on windows
+steps:
+- ${{ if eq(parameters.OS, 'unix')  }}:
+  - script: |
+      curl -sL https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh -o install.sh
+      bash install.sh
+      export NVM_DIR="$HOME/.nvm"
+      
+      # crude version selector since nvm doesn't support it
+      nodeVersion=`echo ${{ parameters.NODE_VERSION }} | sed -e 's/\.x$//g'`
```

Comment:
So this sed removes a ".x" on the end, but I couldn't figure out how that would end up on the version numbers. Did I misunderstand the sed here, or maybe just don't understand how the version numbers end up in this parameter?